main/main.c

```diff
@@ -2255,6 +2255,7 @@ int php_module_startup(sapi_module_struct *sf, zend_module_entry *additional_mod
 
 	zuv.html_errors = 1;
 	zuv.import_use_extension = ".php";
+	zuv.import_use_extension_length = (uint)strlen(zuv.import_use_extension);
```

Comment:
The reason I did what I did, is because the only other usage of zuv.iport_use_extension also uses the strlen:

./Zend/zend.c:800:	zend_uv.import_use_extension_length = (uint)strlen(zend_uv.import_use_extension);

Up to you/devs as to how it should be done, though.